core/src/main/java/org/nervos/ckb/methods/type/transaction/Transaction.java

```diff
@@ -105,6 +108,6 @@ public Transaction sign(BigInteger privateKey) {
       signedWitnesses.add(witness);
     }
     return new Transaction(
-        version, txHash, cellDeps, headerDeps, inputs, outputs, outputsData, signedWitnesses);
+        version, cellDeps, headerDeps, inputs, outputs, outputsData, signedWitnesses);
```

Comment:
I believe it's OK to have `txHash` from the signed tx - sometimes it may be necessary if people want to sign offline and send later.

Perhaps only strip `txHash` when serialized as JSON and sent to RPC.

Reply:
Fixed. https://github.com/nervosnetwork/ckb-sdk-java/pull/162/commits/0f4d0f9d2960c8536d5eccc693fdefd8caa9776c